### PR TITLE
Feat/package

### PR DESCRIPTION
**Pull request by mikemhenry**
This isn't ready to merge, but I just wanted to create the PR so you could see how it looks as I work on it

### CODE REVIEW

File: sc2gameLobby/__init__.py [hunk 1]
```diff
@@ -34,15 +34,16 @@
...
 ################################################################################
-config  = gameConfig.Config         # set up the player/match environment as desired
-host    = hostGame.run              # host a new Starcraft2 match, given a configuration
-join    = joinGame.playerJoin       # join an existing Starcraft2 match, given a configuration
-clear   = gameConfig.clearConfigs   # manually reset/clear any existing (defunct) game configurations
-active  = gameConfig.activeConfigs  # display the games currently being set up
+config = gameConfig.Config  # set up the player/match environment as desired
+host = hostGame.run  # host a new Starcraft2 match, given a configuration
+join = joinGame.playerJoin  # join an existing Starcraft2 match, given a configuration
+clear = (
```
Comment (ttinies) [inline sc2gameLobby/__init__.py]: I don't understand the point of these changes to __init__.py.

I see whitespace / cosmetic changes.
The change to clear doesn't actually change anything.

This __init__ has stuff in here that's defunct and not used anymore and should be removed.  But before that happens, I wanted to confirm that I'm not missing something behind your intent.

**Comment by ttinies** [comment]
I just updated the pretty-ized code that allows multiple clients to play against each other.  It would be good if you could merge in those changes with your renames into the sc2gameLobby subfolder.  Once that's in and you help me understand __init__ changes, I can pull these changes.

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+VERSION = None
+
+# What packages are required for this module to be executed?
+REQUIRED = [
+    "six",
+    "pySC2",
+    "s2clientprotocol",
+    "portpicker"
+    # sc2common,
+    # sc2ladderMgmt,
```
Comment (ttinies) [inline setup.py:30]: FYI, this sc2ladderMgmt repo already exists (just not as a package yet)

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+    # sc2players
```
Comment (ttinies) [inline setup.py:31]: FYI, this sc2players repo already exists (just not as a package yet)

File: sc2gameLobby/__init__.py [hunk 4]
```diff
@@ -18,8 +18,8 @@
 """
 
 from __future__ import absolute_import
-from __future__ import division       # python 2/3 compatibility
```
Comment (mikemhenry) [inline sc2gameLobby/__init__.py:21]: 2 spaces before inline comment

File: sc2gameLobby/__init__.py [hunk 5]
```diff
@@ -32,17 +32,13 @@
 from sc2gameLobby import replay
 from sc2gameLobby import versions
 
+config = gameConfig.Config  # set up the player/match environment as desired
+host = hostGame.run  # host a new Starcraft2 match, given a configuration
+join = joinGame.playerJoin  # join an existing Starcraft2 match, given a configuration
+clear = gameConfig.clearConfigs  # manually reset/clear any existing (defunct) game configurations
+active = gameConfig.activeConfigs  # display the games currently being set up
 
-################################################################################
```
Comment (mikemhenry) [inline sc2gameLobby/__init__.py:36]: 1 space around `=`

File: sc2gameLobby/__init__.py [hunk 6]
```diff
@@ -32,17 +32,13 @@
...
-join    = joinGame.playerJoin       # join an existing Starcraft2 match, given a configuration
-clear   = gameConfig.clearConfigs   # manually reset/clear any existing (defunct) game configurations
-active  = gameConfig.activeConfigs  # display the games currently being set up
 
-
-################################################################################
 def updateVersion(**kwargs):
     """add/update record data using kwargs params for new keys/values"""
     versions.handle.save(kwargs)
-
```
Comment (mikemhenry) [inline sc2gameLobby/__init__.py:48]: Removed trailing newline

**Comment by mikemhenry** [comment]
I've merged in master and explained the changes I did in `__init__.py`

**Review by ttinies — APPROVED** [review]

**Pull request merged**